manifest.json

```
@@ -604,6 +604,14 @@
       "type": "library",
       "version": "22.0.1"
     },
+    {
+      "id": "keyhud",
+      "description": "Simple *key HUD* for Lite XL",
+      "mod_version": "3",
+      "remote": "https://codeberg.org/Mandarancio/keyhud:984f61c56e409ad62b92dff6fa23f399f761d9ab",
```

You also need to update the commit hash, or `lpm` will try to use the old version with the missing comma:
```suggestion
      "remote": "https://codeberg.org/Mandarancio/keyhud:1560d11fb5485aea264ea742e7ee6cd7693f03e0",
```